### PR TITLE
Bump v0.15.0 rc.1

### PR DESCRIPTION
**Pull request by elezar**

### CODE REVIEW

**Review by ArangoGutierrez — APPROVED** [review]

File: deployments/static/extensions-v1beta1-nvidia-device-plugin.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
       # See https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/
       priorityClassName: "system-node-critical"
       containers:
-      - image: nvcr.io/nvidia/k8s-device-plugin:v0.14.4
+      - image: nvcr.io/nvidia/k8s-device-plugin:v0.15.0-rc.1
```
Comment (klueska) [inline deployments/static/extensions-v1beta1-nvidia-device-plugin.yml:38]: As part of "Remove `legacyDaemonsetAPI` config option. This was only required for k8s versions < 1.16." I would have just dropped this whole file.

**Review by klueska — APPROVED** [review]
Small comment but can do in prep for next RC

File: deployments/static/extensions-v1beta1-nvidia-device-plugin.yml (hunk 1 above)
Comment (elezar) [reply deployments/static/extensions-v1beta1-nvidia-device-plugin.yml:38]: Which file?

Comment (klueska) [reply deployments/static/extensions-v1beta1-nvidia-device-plugin.yml:38]: this whole file `deployments/static/extensions-v1beta1-nvidia-device-plugin.yml`

Comment (elezar) [reply deployments/static/extensions-v1beta1-nvidia-device-plugin.yml:38]: OK. Will do as a cleanup. Thanks.

**Pull request merged**